src/package.xml

```diff
@@ -2324,9 +2327,23 @@
         <name>EmailTemplate</name>
     </types>
     <types>
+        <members>HasUserManagedTDTM</members>
+        <members>IsEnabled_AddressVerification</members>
+        <members>IsEnabled_AutomaticPayments</members>
         <members>IsEnabled_CustomizableRollups</members>
+        <members>IsEnabled_DefaultGAU</members>
+        <members>IsEnabled_HouseholdAcctModel</members>
+        <members>Org_IsEnabled_AdvancedMultiCurrency</members>
```

Comment:
What is naming convention for the `FeatureParameterBoolean`?

Reply:
There isn't necessarily a naming convention in place. I've tried to stick to something somewhat logical based on the type of data (IsEnabled, Org_IsEnabled, Data, etc.), but to be honest it's not really visible anywhere other than our code.